### DISCUSSION TITLE
Using the OAuth2 Server Bundle with Pimcore error

### DISCUSSION THREAD

**Discussion by MyZik** [pimcore/pimcore#11360] [Q&A / Support] [answered]
Hello. I'm trying to implement the OAuth2 support in my Pimcore project, with this Symfony Bundle: https://github.com/thephpleague/oauth2-server-bundle

At the step 5 in (https://github.com/thephpleague/oauth2-server-bundle/blob/master/docs/index.md) we have:

> Enable the authenticator security system in config/security.yaml file:
> 
> ```yaml
> security:
>     enable_authenticator_manager: true
> ```

If we do that in Pimcore security.yaml, we will get an error:

> The "security.firewall.pimcore_admin.guard.entry_point" option has no effect in the new authenticator system, configure "security.firewall.pimcore_admin.entry_point" instead.

Moving option `entry_point` from `security.firewall.pimcore_admin.guard` to `security.firewall.pimcore_admin` breaks the Pimcore authenticators:

> The authenticator manager no longer has "anonymous" security. Please remove this option under the "pimcore_admin" firewall.

How can I use the bundle with Pimcore correctly?

**Comment by dvesh3** [comment, marked as answer]
@MyZik Hi, this is based on new security authentication introduced with Symfony 5.1, which Pimcore does not support at the moment. So you have to wait until this PR https://github.com/pimcore/pimcore/pull/10967 is merged.

**Reply by MyZik** [reply]
Hi @dvesh3 thank you for info!
